src/app/community/_components/PostCardDetailModal/PostCardDetailModal.tsx

```diff
@@ -143,35 +183,27 @@ export default function PostCardDetailModal({ cardId, onClose, isMine, commentCo
   useEffect(() => {
     // 처음에 가져온 댓글 데이터
     queryClient.setQueryData(['infiniteCommentData'], null);
-    if (data?.data) {
-      const initialComments = data.data.comments;
+    if (postCardListData?.data) {
+      const initialComments = postCardListData.data.comments;
       const lastCommentData = initialComments[initialComments.length - 1];
       setVisibleComments(initialComments);
       setLastCommentId(lastCommentData?.id);
     }
-  }, [data, queryClient]);
+  }, [postCardListData, queryClient]);
 
-  if (!data) return <ModalSkeleton />;
+  if (!postCardListData) return <ModalSkeleton />;
 
-  const { data: postData, status, message } = data;
+  const { data: postData, status, message } = postCardListData;
 
-  if (status === 'ERROR' || postData === null) {
+  if (status === 'ERROR') {
     toast.error(message);
     onClose();
     return null;
   }
 
   const { content, likeCount, nickName, reviewImages, title, updatedAt, userImage, custom, isLiked } = postData;
```

Comment:
```suggestion
  const { content, commentCount, likeCount, nickName, reviewImages, title, updatedAt, userId, userImage, custom, isLiked } = postData;
```
userId랑 commentCount는 props로 전달하지 않고 그냥 postData에서 가져와도 되지 않나요?

Reply:
요거슨 도원님 pr 머지되면 추가수정 들어갈게요!